dev/restinio/buffers.hpp

```diff
@@ -534,7 +534,7 @@ class writable_item_t
 		}
 
 		writable_item_t &
-		operator = ( writable_item_t && b )
+		operator = ( writable_item_t && b ) noexcept
```

Comment:
There is the following implementation of move-operator:
```cpp
writable_item_t &
operator = ( writable_item_t && b )
{
	if( this != &b )
	{
		destroy_stored_buffer();
		m_write_type = b.m_write_type;
		b.get_writable_base()->relocate_to( &m_storage );
	}

	return *this;
}
```
Method `relocate_to` is not marked as `noexcept` in `writable_base_t`. It means that it can throw. And because of that move-operator can't be marked as `noexcept`.

Reply:
Makes sense. It looks like clang-tidy could not analyze deep enough to consider the implementation of relocate_to.